node/network/bridge/src/lib.rs

```diff
@@ -290,15 +290,15 @@ where
 			},
 
 			Action::ReportPeer(peer, rep) => {
-				tracing::debug!(
+				tracing::trace!(
```

Comment:
I'd left this as `debug`. There should not be that many misbehaving peers

Reply:
Actually that is also about rewarding peers. I think it is still relatively spammy.

Reply:
we usually log it at a higher level so that it has more precise context wrt LOG_TARGET
grep async fn modify_reputation

Reply:
I made it debug again anyway.